src/User_Command.php

```diff
@@ -722,27 +730,35 @@ public function add_role( $args, $assoc_args ) {
 	 * <user>
 	 * : User ID, user email, or user login.
 	 *
-	 * [<role>]
-	 * : A specific role to remove.
+	 * [<role>...]
+	 * : Remove the specified role(s) from the user.
 	 *
 	 * ## EXAMPLES
 	 *
 	 *     $ wp user remove-role 12 author
 	 *     Success: Removed 'author' role for johndoe (12).
 	 *
+	 *     $ wp user remove-role 12 author editor
+	 *     Success: Removed 'author' role for johndoe (12).
+	 *     Success: Removed 'editor' role for johndoe (12).
+	 *
 	 * @subcommand remove-role
 	 */
 	public function remove_role( $args, $assoc_args ) {
 		$user = $this->fetcher->get_check( $args[0] );
 
 		if ( isset( $args[1] ) ) {
-			$role = $args[1];
+			$roles = $args;
```

Comment:
Same comment about erroring early if no role is provided.

Reply:
@danielbachhuber No roles on the remove-role command is for multisite installs to remove the user from a blog.

Reply:
@shawnhooper Ah, good point.